Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -267,6 +267,14 @@ def _parse_demisto_comments(ioc: dict, comment_field_name: str, comments_as_tags
             return None
         return [comment]
 
+    elif comment_field_name == 'indicator_link':
+        # parse indicator link into tags
+        raw_id = ioc.get('id')
+        if raw_id:
+            return f'https://{demisto.demistoUrls()}/#/indicator/{raw_id}'.split(",")
```

Comment:
1. so it will return a list as needed in _parse_demisto_comments. I change it to [f'https://{demisto.demistoUrls()}/#/indicator/{raw_id}']- is it ok @eyalpalo ?
2. I added it just in case, but I'll remove it